docs/reference/query-dsl/percolator-query.asciidoc

```diff
@@ -0,0 +1,426 @@
+[[query-dsl-percolator-query]]
+=== Percolator Query
+
+Traditionally you design documents based on your data, store them into an index, and then define queries via the search API
+in order to retrieve these documents. The percolator works in the opposite direction. First you store queries into an
+index and then, via the `percolator` query, you define documents in order to retrieve these queries.
```

Comment:
and then you use the `percolator` query to search for the queries which match a specified document (or documents).